### ISSUE TITLE
Crash when using compose v `1.8.0-alpha06` on android

### ISSUE THREAD

**Issue by TepesLucian** [composablehorizons/compose-unstyled#44]
Code
```
ModalBottomSheet(
        state = rememberModalBottomSheetState(initialDetent = SheetDetent.FullyExpanded),
    ) {
        Sheet(
            modifier = Modifier
                .background(Color.Red)
                .fillMaxSize(),
        ) {

        }
    }
```
Crash:
```
java.lang.NoSuchMethodError: No direct method <init>(Ljava/lang/Object;Lkotlin/jvm/functions/Function1;Lkotlin/jvm/functions/Function0;Landroidx/compose/animation/core/AnimationSpec;Landroidx/compose/animation/core/DecayAnimationSpec;Lkotlin/jvm/functions/Function1;ILkotlin/jvm/internal/DefaultConstructorMarker;)V in class Landroidx/compose/foundation/gestures/AnchoredDraggableState; or its super classes (declaration of 'androidx.compose.foundation.gestures.AnchoredDraggableState' appears in /data/app/~~oNy348mAOgdSFPflHFliag==/com.x.y.z-LrJmhEfwZDuVcA0oX9CIrw==/base.apk)
    	at com.composables.core.BottomSheetState.<init>(BottomSheet.kt:160)
    	at com.composables.core.BottomSheetKt.rememberBottomSheetState$lambda$17(BottomSheet.kt:85)
    	at com.composables.core.BottomSheetKt.$r8$lambda$HOlp03M64Z9EpLpUe6obdQw4518(Unknown Source:0)
    	at com.composables.core.BottomSheetKt$$ExternalSyntheticLambda12.invoke(D8$$SyntheticClass:0)
```
I've switched to stable version of compose and eveything works as it should.

**Comment by alexstyl** [comment]
Needs more info. Which platform and compose version is the crash from?

**Comment by TepesLucian** [comment]
Android platform with compose `androidx.compose:compose-bom-alpha:2024.11.00` bom version.

**Comment by alexstyl** [comment]
Was able to reproduce. Investigating

**Comment by alexstyl** [comment]
Fixed. Fix will be available on the next release

**Comment by alexstyl** [comment]
This is now available in [1.20.0](https://github.com/composablehorizons/compose-unstyled/releases/tag/1.20.0)
